grub/pkg.yaml

```diff
@@ -64,6 +64,12 @@ steps:
         mkdir -p /rootfs/usr/share/locale
         mkdir -p /rootfs/usr/share/bash-completion/completions
         mv -v /rootfs/etc/bash_completion.d/grub /rootfs/usr/share/bash-completion/completions
+
+        mv /rootfs/usr/sbin/* /rootfs/usr/bin/
```

Comment:
Maybe there's an option in configure (or a make option) to tell it where to place the binaries. It'd be better, but unsure if one exists